src/ISA-XLSX/ISA-XLSX.IO.fs

```diff
@@ -374,16 +379,16 @@ module ISA_Investigation  =
     /// Creates an empty ivestigation file at the given path
     let createEmpty path (investigation : InvestigationItem) = 
```

Comment:
Maybe one could input the sheetName as a param instead of it being fixed. Might be a thing for the config